x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/index/mapper/GeoShapeWithDocValuesFieldMapper.java

```diff
@@ -482,6 +482,11 @@ protected void checkIncomingMergeType(FieldMapper mergeWith) {
         super.checkIncomingMergeType(mergeWith);
     }
 
+    @Override
+    protected SyntheticSourceMode syntheticSourceMode() {
```

Comment:
Note that it should be relatively easy to natively support synthetic source for `GeoShapeWithDocValuesFieldMapper`, because this field mapper can store shapes in an accurate and more efficient format as well. We should then default the `store` attribute to `true`, like we did for other field types as well. We can do this in a follow up.

Reply:
One benefit of current implementation is that it covers `ignore_malformed` automatically. I'll create an issue and we can prioritize it based on usage.

Reply:
#108980

Reply:
👍